packages/firestore/lib/index.d.ts

```diff
@@ -2084,7 +2086,7 @@ export namespace FirebaseFirestoreTypes {
      *
      * #### Example
      *
-     * ```js
+     * ```jsc
```

Comment:
I don't understand this change, it does not seem relevant to the PR, and it does not seem to be in the [languages.yaml](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) file anywhere I would expect for javascript highlighting in [the package github uses](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) ?

Reply:
Sorry, I missed this change. This isn't intended one(maybe from linter?). I'll remove this.